src/ws/session.c

```diff
@@ -62,6 +62,7 @@
 #define DEBUG_SESSION 0
 #define EX 127
 #define DEFAULT_PATH "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+#define DEFAULT_KTAB PACKAGE_SYSCONF_DIR "/cockpit/krb5.keytab"
```

Comment:
This is not the *default* ktab, could this be called `COCKPIT_KTAB`?

Reply:
It's our default ktab, but sure